src/main/java/com/exsol/errorcodemodel/ErrorIdentifier.java

```diff
@@ -27,7 +26,56 @@ public static ErrorIdentifier parse(final String errorCode) throws SyntaxExcepti
     }
 
     /**
-     * Possible types of exasol error codes.
+     * Create a new instance of an {@link ErrorIdentifier}.
+     * @param type type of error (warning, error, fatal)
```

Comment:
```suggestion
     * @param type type of error
```
This is an enum. Is it necessary to list the options here?

Reply:
Since `type` was chosen as name, that is not very self explaining. That's why I put in examples.